src/AbstractDbEntity.php

```diff
@@ -498,7 +498,7 @@ public function getModifiedDbProperties()
      */
     public function clearModifiedDbProperty($property)
     {
-        if (($key = array_search($property, $this->modifiedDbProperties))) {
+        if (($key = array_search($property, $this->modifiedDbProperties)) !== false) {
```

Comment:
I don't really see why this would be needed and what it would improve?

Reply:
modified db values are added to the internal `$modifiedDbProperties` array. First property in gets the index `0`

https://github.com/starweb/starlit-db/blob/cad5d41975af69915a9dd638459a6a245bcf435c/src/AbstractDbEntity.php#L336-L338

If you want to clear  that property using above function the loose type comparison will be a `if (0)` check which will return false thus not clearing that value.

Reply:
aaah excellent. Now I see 🙂